index.html

```diff
@@ -38,80 +38,87 @@ <h1 class="heading">I'm Denyse. Glad to see you!</h1>
-                    <li>
                         <ul class="languages languages-1">
-                            <li>HTML</li>
-                            <li>CSS</li>
+                            <li>html</li>
+                            <li>css</li>
                             <li>Javascript</li>
                         </ul>
-                    </li>
                     <button class="project-1">See project</button>
-            </li>
-            <li class="container2">
+                </li>
+                </ul>
+            <ul class="container2 containery">
+                <li>
                     <div class="works-pic"><img src="Assets/files/Snapshoot2.png" alt=""></div>
-                    <div class="card2"><img src="Assets/files/Snapshoot desktop2.jpg" alt=""></div>
+                    <div class="responsive-image2"><img src="Assets/files/Snapshoot desktop2.jpg" alt="Responsive Image"></div>
+                </li>
+                <li>
                     <h2 class="works-heading deskheading2">Multi-Post Stories</h2>
                     <div class="canopy canopy2 canopyX"><img src="Assets/files/Frame 2.png" alt=""></div>
                     <div class="canopyX10"><img src="Assets/files/Frame facebook.jpg" alt=""></div>
                     <p class="works-paragraph deskpar2 deskparY">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
                     <p class="deskpar3">Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.</p>
-                    <li>
                         <ul class="languages languages-2">
-                            <li>HTML</li>
+                            <li>html</li>
                             <li class="ruby">Ruby on rails</li>
-                            <li>CSS</li>
+                            <li>css</li>
                             <li>Javascript</li>
                         </ul>
-                    </li>
                     <button class="project-2">See project</button>
-            </li>
-            <li class="container1 container3">
+                </li>
+            </ul>
+            <ul class="container1 containerx">
+                <li>
                     <div class="works-pic"><img src="Assets/files/Snapshoot3.png" alt=""></div>
-                    <div class="card1"><img src="Assets/files/Snapshoot desktop Portfolio3.jpg" alt=""></div>
+                    <div class="responsive-image1"><img src="Assets/files/Snapshoot desktop Portfolio3.jpg" alt="Responsive Image"></div>
+                </li>
+                <li>
                     <h2 class="works-heading tonic">Tonic</h2>
                     <h2 class="works-heading facebook">Facebook 360</h2>
                     <div class="canopy canopy1 canopyX"><img src="Assets/files/Frame 2.png" alt=""></div>
```

Comment:
- [ ] Please as mentioned by the previous reviewer, Do not use images for the experience list. Please use list tags for the experience list (e.g. canopy :white_circle: facebook :white_circle: 2015). It is not supposed to be an image. The image is not responsive, also it's a bit inconsistent and looks poor quality. :slightly_smiling_face: